components/automate-ui/src/app/entities/layout/layout.facade.ts

```diff
@@ -11,23 +12,31 @@ import { MenuItemGroup } from './layout.model';
 import { sidebarMenuGroups, showPageLoading } from './layout.selectors';
 import { ShowPageLoading, UpdateSidebarMenuGroups } from './layout.actions';
 
+import { Project } from 'app/entities/projects/project.model';
+import { allProjects } from 'app/entities/projects/project.selectors';
+
 // Important! These must match components/automate-ui/src/styles/_variables.scss
 enum Height {
-  Navigation = '70px',
-  Banner = '110px'
+  Navigation = 70,
+  Banner = 30,
+  ProcessProgressBar = 48,
+  PendingEditsBar = 36
 }
 
 @Injectable({
   providedIn: 'root'
 })
 export class LayoutFacadeService {
-  headerHeight = '70px';
-  contentHeight = `calc(100% - ${this.headerHeight})`;
+  // headerHeight = '70px';
```

Comment:
remove these?

Reply:
@bcmdarroch yes, i will. I left a note in the PR that I'm not finished with the layout height adjustments for the bottom user notification bars.

Reply:
ah sorry missed that!